main.tex

```diff
@@ -11,6 +11,7 @@
 
 \usepackage{cmap}				% Mapear caracteres especiais no PDF
 \usepackage[utf8]{inputenc} 			% utf8 definition
+\usepackage[T1]{fontenc}                        % Isso arruma o problema de copiar/colar do PDF em alguns programas. Tbm dá pra usar '<' e '>'.
```

Comment:
There is a small trailing space in there
```suggestion
\usepackage[T1]{fontenc}                       % Isso arruma o problema de copiar/colar do PDF em alguns programas. Tbm dá pra usar '<' e '>'.
```

Reply:
Somehow the github visualizer its showing like there is a space there, my mistake